### DISCUSSION TITLE
Mini EV: Two Years On - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#53] [General]
# Mini EV: Two Years On - Alan Pope's blog

tl;dr I have had a Mini EV for a little over two years, so I thought it was time for a retrospective. This isn’t so much a review, as I’m not a car journalist. It’s more just my thoughts of owning an electric car for a couple of years.
I briefly talked about the car in episode 24 (out on Tuesday 5th march) of Linux Matters Podcast, if you prefer a shorter, less detailed review in audio format.

https://popey.com/blog/2024/03/mini-electric-two-years-on/

**Comment by chrisjrob** [comment]
Great post, lovely to hear some positives about EVs. Just a minor point about the lack of a spare tyre, this affects most new cars these days. The weight reduction improves economy.

**Reply by richardcrossley** [reply]
Good to read. Personally don't own a car anymore, just not required at this time, but I am intrigued to hear about EV ownership.
